presto-common/src/main/java/com/facebook/presto/common/ViewDefinition.java

```diff
@@ -11,18 +11,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.metadata;
+package com.facebook.presto.common;
```

Comment:
Do you need `ViewDefinition` in presto-analyzer or presto-common? I guess presto-analyzer is good enough? presto-common will expose too much information to all other modules.

Reply:
yeah, it seems presto-analyzer is the better place.